src/Main.java

```diff
@@ -10,30 +10,22 @@ public static void main(String[] args) {
 
         /// Задание №2.
         int yearPhon = 2013;
-        if (clientOS == 0) {
-            if (yearPhon < 2015) {
-                System.out.println("Установите облегченную версию приложения для iOS по ссылке");
-            } else {
-                System.out.println("Установите версию приложения для iOS по ссылке");
-            }
+        if (clientOS == 0 && yearPhon < 2015) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else if (clientOS == 0 && yearPhon > 2015) {
```

Comment:
```suggestion
        } else if (clientOS == 0 && yearPhon >= 2015) {
```
если год будет 2015, то ничего не выведется - надо включить 2015